sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -446,4 +452,84 @@ export class InstitutionController extends BaseController {
       },
     }));
   }
+
+  /**
+   * Get the Institution details for the ministry institution detail page
+   * @param institutionId
+   * @returns AESTInstitutionDetailDto
+   */
+  @AllowAuthorizedParty(AuthorizedParties.aest)
+  @Groups(UserGroups.AESTUser)
+  @Get("/:institutionId/detail")
+  async getAESTInstitutionDetailById(
+    @Param("institutionId") institutionId: number,
+  ): Promise<AESTInstitutionDetailDto> {
+    if (!institutionId) {
+      throw new UnprocessableEntityException(
+        "Institution Id is required to fetch the institution detail",
+      );
+    }
+    const institutionDetail =
+      await this.institutionService.getAESTInstitutionDetailById(institutionId);
+    return {
+      legalOperatingName: institutionDetail.legalOperatingName,
+      operatingName: institutionDetail.operatingName,
+      primaryPhone: institutionDetail.primaryPhone,
+      primaryEmail: institutionDetail.primaryEmail,
+      website: institutionDetail.website,
+      regulatingBody: institutionDetail.regulatingBody,
+      institutionTypeName: institutionDetail.institutionType.name,
+      formattedEstablishedDate: dayjs(institutionDetail.establishedDate).format(
```

Comment:
Please move this to the date-utils.ts and create something similar to the _dateString_.